snippets.json

```diff
@@ -0,0 +1,23 @@
+{
+	"insertMarkdownLintDisableComment": {
+		"prefix": "markdownlint-disable",
+		"body": "<!-- markdownlint-disable$1 -->",
```

Comment:
space before the $1?

Reply:
You think so? I was trying to accommodate the case where no rules would be specified, and so the user would just tab out instead of entering anything. In that case, including a space would result in funky-looking formatting (albeit still valid code): `<!-- markdownlint-disable  -->`.

Although, now that you bring it up, I just tested whether you could backspace from an empty tab stop without losing the position (i.e. after `<!-- markdownlint-disable $1 -->` is inserted, hit `[backspace]` with the result that `<!-- markdown-lint$1 -->` so you could still then still tab out to the end of the snippet `<!-- markdown-lint -->|` vs. `<!-- markdown-lint| -->`), and it looks like you can. So I will go ahead add the space before.

Reply:
I was wondering if it was going to stick the MD### to the text was all. Not sure how the UI for this works

Reply:
With the previous format (`<!-- markdownlint-disable$1 -->`) it would have. I was intending to accommodate the no-rules-specified use case, and to just let users add the space themselves (i.e. `<!-- markdownlint-disable| -->` + `[space]` => `<!-- markdownlint-disable | -->`) if they wanted to list specific rules. However, I think adding a space by default makes sense in light of my previous comment.